### DISCUSSION TITLE
Datetime picker internationalization

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#20436] [Feature Requests]
### Summary

In some language, the first day of the week is Monday

Another feature is to be able to type in the date.
It is quicker to type straight 21/12/2018 than to type in various places 2018 then a drop-down to choose December then click on 21 especially when the user has to find out these places.

### Motivation

To be in accordance with the users

### Detailed Design

<interface-datetime  :start-day-of-week="1" />

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

none

### Adoption Strategy

none

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
